india_compliance/gst_india/overrides/payment_entry.py

```diff
@@ -361,9 +361,6 @@ def get_advance_payment_entries_for_regional(
     company = frappe.db.get_value("Account", party_account, "company")
     taxes = get_taxes_summary(company, payment_entries)
 
-    if not taxes:
-        return
```

Comment:
Instead `return payment_entries` may be correct fix.

Reply:
Previously the code was not there so I reverted back to same but same can also be done.

Reply:
added return payment_entries